spynl/main/mail.py

```diff
@@ -11,8 +11,8 @@
 from jinja2 import Environment, Template, TemplateNotFound, FileSystemLoader
```

Comment:
The dummy mailer is used in the unit tests. But the line at 158 can probably go (I'll put a comment at the line, because I got a bit confused by these comments at the start of the file)